core/server/utils.js

```diff
@@ -51,3 +51,28 @@ export const parseQuery = query => {
     page,
   };
 };
+
+const mergeOptions = (base, custom) => {
+  const options = Object.assign({ ...base }, custom || {});
```

Comment:
@DAreRodz  I don't think you need to spread the object here, that's what is for `Object.assign()`, I think.

Reply:
Nevermind, your solution is right :)